### PR TITLE
[IOPLT-239] Creates the `Stepper` component

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR creates the `Stepper` component

## List of changes proposed in this pull request
- Creates Stepper component
- Include standard example in example App
- Include header integration in example App

## How to test
Check Example app for usage and UI Checking

### CODE REVIEW

File: src/components/stepper/Stepper.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  IOSpacer,
+  IOStyles,
+  IOVisualCostants,
+  useIOExperimentalDesign
+} from "../../core";
+
+type StepperProps = {
+  steps: number;
+  currentStep: number;
+  padded?: boolean;
```
Comment (dmnplb) [inline src/components/stepper/Stepper.tsx:16]:
```suggestion
```
We need to use this component in a fixed position, so we probably always need to add this property. We can take it for granted.

Comment (dmnplb) [inline src/components/stepper/Stepper.tsx:16]:
```suggestion
```
We need to use this component in a fixed position, so we probably always need to add this property. We can take it for granted.

File: src/components/stepper/Stepper.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+const legacyColorMap: Record<string, IOColors> = {
+  active: "blue",
+  inactive: "bluegrey"
+};
+
+export const Stepper = ({
+  steps,
+  currentStep,
+  padded = false
```
Comment (dmnplb) [inline src/components/stepper/Stepper.tsx:33]:
```suggestion
```

File: src/components/stepper/Stepper.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  );
+  const onLayout = (e: LayoutChangeEvent) => {
+    setStepWidth(e.nativeEvent.layout.width / steps);
+  };
+
+  return (
+    <View
+      style={
+        padded ? { paddingHorizontal: IOVisualCostants.appMarginDefault } : {}
+      }
```
Comment (dmnplb) [inline src/components/stepper/Stepper.tsx:52]:
```suggestion
      style={{ paddingHorizontal: IOVisualCostants.appMarginDefault }}
```

**Review by dmnplb — CHANGES_REQUESTED** [review]

**Review by dmnplb — APPROVED** [review]

**Comment by thisisjp** [comment]
@CrisTofani @dmnplb looks like the unactive step has a darker gray compared to the design 👀

**Comment by dmnplb** [comment]
@CrisTofani @thisisjp That's correct, but I missed it completely because the new version (with experimental DS on) has the correct chromatic value. Let's fix it before the next release.

**Pull request merged**